bpf/lib/conntrack.h

```diff
@@ -243,6 +276,9 @@ static __always_inline __u8 __ct_lookup(const void *map, struct __ctx_buff *ctx,
 		return CT_ESTABLISHED;
 	}
 
+#ifdef HAVE_LARGE_INSN_LIMIT
+ct_new:
```

Comment:
Nit: instead of protecting with the ifdef, you could `ct_new __maybe_unused:`

Reply:
This didn't work:
```
  	 In file included from bpf_lxc.c:29:
  	 In file included from /cilium/bpf/lib/lxc.h:16:
  	 In file included from /cilium/bpf/lib/proxy.h:7:
  	 /cilium/bpf/lib/conntrack.h:279:1: error: use of undeclared identifier 'ct_new'
  	 ct_new __maybe_unused:
  	 ^
  	 1 error generated.
  	 make: *** [Makefile:287: bpf_lxc.ll] Error 1
```

Reply:
You want:
```c
ct_new: __maybe_unused
```
See other usage in the code base (e.g., in `bpf/lib/proxy.h`).

Reply:
Done. Thanks @brb , @pchaigno .